packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -36,13 +36,20 @@ export interface FileTriggerProps extends SlotProps, DOMProps, AriaLabelingProps
   onChange?: (files: FileList | null) => void
 }
 
-export const FileTriggerContext = createContext<ContextValue<FileTriggerProps, HTMLDivElement>>(null);
+interface FileTriggerContextValue extends FileTriggerProps {
+  inputRef?: RefObject<HTMLInputElement>
```

Comment:
I think this is a good question for RAC in general about how we want to expose refs for sub-elements. What was the use case for adding this now?

Reply:
@yihuiliao  If I remember correctly from our convo, this `inputRef` was added to support the RSP DropZone's imperativeRef since we wanted to match with how refs are treated in other RSP components (aka TextField), where the ref is attached to the wrapper but users can still access the inner input via `getInputElement`.

However, I wonder we even need to expose a way to get the input element for the FileTrigger. It makes sense for TextField since people often do stuff with the input element directly (aka read the value), but what about the FileTrigger's input? Will people need to access to it?

Reply:
Yup, that's correct. Personally, I'm with Daniel in that I'm not really sure we need to expose the input element for FileTrigger but there could be use cases that I might be missing.